app/models/pillar.rb

```diff
@@ -44,17 +44,18 @@ def value(pillar:)
 
     def all_pillars
       {
-        company_name: "certificate_information:subject_properties:O",
-        company_unit: "certificate_information:subject_properties:OU",
-        email:        "certificate_information:subject_properties:Email",
-        country:      "certificate_information:subject_properties:C",
-        state:        "certificate_information:subject_properties:ST",
-        city:         "certificate_information:subject_properties:L",
-        dashboard:    "dashboard",
-        apiserver:    "api:server:external_fqdn",
-        http_proxy:   "proxy:http",
-        https_proxy:  "proxy:https",
-        no_proxy:     "proxy:no_proxy"
+        company_name:     "certificate_information:subject_properties:O",
+        company_unit:     "certificate_information:subject_properties:OU",
+        email:            "certificate_information:subject_properties:Email",
+        country:          "certificate_information:subject_properties:C",
+        state:            "certificate_information:subject_properties:ST",
+        city:             "certificate_information:subject_properties:L",
```

Comment:
JFYI there will be some collissions with https://github.com/kubic-project/velum/pull/210 - so one or the other will need a rebase after a merge